docs/example-resource.md

```diff
@@ -0,0 +1,50 @@
+---
+title: INSERT-TITLE
+layout: /markdown_resource.html.erb
+---
+
+# INSERT-TITLE
```

Comment:
question from a n00b - does it matter if there's a space between the hash and the title or not?

Reply:
omg that sucks, ive been doing that completely wrong :(

Reply:
Actually scrap that, it seems to render correctly still! I think it's just best practice to add spacing: https://google.github.io/styleguide/docguide/style.html#add-spacing-to-headings